cnd/src/swap_protocols/rfc003/messages.rs

```diff
@@ -48,6 +58,12 @@ pub struct Decline {
     pub reason: Option<SwapDeclineReason>,
 }
 
+impl Scribe for Decline {
+    fn scribe(&self) -> String {
+        format!("Declining swap: {}", self.swap_id.scribe())
```

Comment:
This is something that I'd rather have inline with the actual processing logic. IMO, traits like `Display` etc should not assume how a struct is used because they cannot possibly know.

Reply:
The `Scribe` (for want of a better name) trait is _only_ used for logging so it seems reasonable to assume how it is used?